### ISSUE TITLE
📦 in SwiftPkgUpdates tweets seems to break twitter search

### ISSUE THREAD

**Issue by finestructure** [SwiftPackageIndex/SwiftPackageIndex-Server#1384]
I cannot find any tweets of new packages. For instance:

<img width="476" alt="CleanShot 2021-11-19 at 05 39 56@2x" src="https://user-images.githubusercontent.com/65520/142565910-5b2068b1-b1a5-477f-a152-824148ccea36.png">

But also searching for a known string in a "new package" tweet like the following

<img width="397" alt="CleanShot 2021-11-19 at 05 40 25@2x" src="https://user-images.githubusercontent.com/65520/142565940-094cefc3-34f7-4106-bc87-a144d4b8f4fe.png">
 
doesn't return any results:

<img width="473" alt="CleanShot 2021-11-19 at 05 42 05@2x" src="https://user-images.githubusercontent.com/65520/142566090-bf99c960-8dd7-481a-b2f5-bda26aecad0a.png">

I suspect this is due to the `📦` character.

**Comment by daveverwer** [comment]
Secretly, I've never been a fan of using these emojis in this way. Not enough to do anything about it, but if there's an excuse to remove them I'm all in! 😂

**Comment by finestructure** [comment]
It's bizarre, it _does_ work, sort of, just not with the "new package tweets":

<img width="629" alt="CleanShot 2021-12-01 at 12 04 02@2x" src="https://user-images.githubusercontent.com/65520/144223538-bc1eb557-f90a-4d5b-8e41-a75114613067.png">

I've tried searching for the "new" emoji 🆕 (to see if it'd be a valid replacement) and that just fails outright:

<img width="520" alt="CleanShot 2021-12-01 at 12 05 49@2x" src="https://user-images.githubusercontent.com/65520/144223722-5a1da208-f1a0-47b6-91de-57e8d0da13f7.png">

Options are:

1. replace 📦 with 🚀 - I've tested a search and that seems to work with 🚀 as the first character
2. leave 📦 and just live with the search issue
3. replace 📦 with `NEW:` (and ⬆️ with `UPDATE`)
4. replace 📦 or `New:` (and ⬆️ with `Update:`)

I dislike 3 and 4 because they take away available characters from the package info unnecessarily. If we went with `New` / `Update`, we should at least drop the `just released` and `just added` from the rest of the tweet.

I do prefer the emojis over plain text because makes the tweets a little less bland but mostly because I can very easily see at a glance if it's a new package or an update.

My order of preference would be: 1, 2, gap, 4, 3.

**Comment by daveverwer** [comment]
Secret option number 5... None of the above. All of the information is in the tweet already:

* "Apple **just released** swift-nio-http2 v1.19.0 – HTTP/2 support for SwiftNIO"
* "BrightDigit **just added** a new package, Swiftube"

**Comment by finestructure** [comment]
That's hard to parse even with the bolding, which we won't have in tweets... Why make it so hard to digest? :)

**Comment by daveverwer** [comment]
I'll leave the choice to you then. My vote would be for just the plain tweet like I suggested but that's due to my dislike of using emojis in this way.

**Comment by finestructure** [comment]
Door number 2 it is then. Boom, another one bites the dust 🤣
